pkg/collector/corechecks/snmp/internal/devicecheck/devicecheck.go

```diff
@@ -150,6 +155,23 @@ func (d *DeviceCheck) Run(collectionTime time.Time) error {
 		d.sender.ReportMetrics(d.config.Metrics, values, tags)
 	}
 
+	// Get a system appropriate ping check
+	if d.config.PingEnabled {
+		log.Infof("ATTEMPTING TO RUN PING FOR HOST: %s, tags: %+v", d.config.IPAddress, tags)
+		pingResult, err := d.Ping(d.config.PingConfig)
+		if err != nil {
+			log.Errorf("%s: failed to ping device: %s", d.config.IPAddress, err.Error())
+			d.sender.Gauge(pingCanConnectMetric, float64(0.0), tags)
+		} else {
+			log.Infof("%s: ping returned: %+v", d.config.IPAddress, pingResult)
+			d.sender.Gauge(pingAvgRttMetric, float64(pingResult.AvgRtt/time.Millisecond), tags)
+			d.sender.Gauge(pingCanConnectMetric, common.BoolToFloat64(pingResult.CanConnect), tags)
+			d.sender.Gauge(pingPacketLoss, pingResult.PacketLoss, tags)
+		}
+	} else {
+		log.Infof("PING DISABLED FOR HOST: %s, tags: %+v, ping enabled: %t, ping config: %+v", d.config.IPAddress, tags, d.config.PingEnabled, d.config.PingConfig)
```

Comment:
1/ I think we shouldn't use Info log level since this is expected behaviour. Is that for developement debugging purpose?

2/ Maybe we should avoid using UPPER CASE if not necessary?